lab-matt/data/model/123456

```diff
@@ -0,0 +1 @@
+{"id":123456,"name":"test"}
```

Comment:
This is okay for now, I suggest making your test data reside within your test folder in entirety in the future :)